feat: add instruct wrapper (#1768)
embeddings-benchmark/mteb: docs/adding_a_model.md

    ###### Instantiating the Model with Prompts

    If you are unable to directly add the prompts in the model configuration, you can instantiate the model using the `sentence_transformers_loader` and pass `prompts` as an argument. For more details, see the `mteb/models/bge_models.py` file.

##### Adding instruction models

> Model that using instructions during, then it can use [`InstructSentenceTransformerWrapper`](../mteb/models/instruct_wrapper.py).

Reviewer: ```suggestion
Models that use instructions can use the [`InstructSentenceTransformerWrapper`](../mteb/models/instruct_wrapper.py). For example:
```
